ossdbtoolsservice/utils/telemetryUtils.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ossdbtoolsservice.serialization import Serializable
+from ossdbtoolsservice.utils import constants
+
+class TelemetryErrorParams(Serializable):
+    """Parameters to be sent back with a telemetry event"""
+
+    def __init__(self, properties: dict()):
+        properties['providerName'] = 'MySQL'
```

Comment:
Events coming from your extension will always start with "azuredatastudio-mysql" - so you don't really need to add the provider since you'll know that it's always going to be MySQL. 

(it doesn't technically hurt to have it, but it just seems unnecessary to me)